packages/core/src/odata-common/header-builder.spec.ts

```diff
@@ -112,63 +109,6 @@ describe('Header-Builder', () => {
     expect(headers['SAP-Connectivity-SCC-Location_ID']).toBe('Potsdam');
   });
 
-  it('Adds location id headers to the fetch csrf token request if there is a cloudConnectorLocationId in the destination', async () => {
```

Comment:
Have these tests moved or are they obsolete?

Reply:
I re-wrote the tests.